### ISSUE TITLE
Erroneous error: "The current account does not have access to any namespaces."

### ISSUE THREAD

**Issue by mecampbellsoup** [kubeapps/kubeapps#2980]
### Description:

We are using kubeapps with an auth proxy in front of it.

When I visit the local domain `https://apps.coreweave.test/#/` the request to `/api/clusters/default` always fails and this warning is displayed.

![image](https://user-images.githubusercontent.com/2043821/122118203-c6beb280-cdf5-11eb-8af5-bc032e9e70c2.png)

Looking at the [frontend code](https://github.com/kubeapps/kubeapps/blob/master/dashboard/src/actions/namespace.ts#L52-L76), it does not seem like this error can be avoided as our kubeapps "users" can only view their own namespace, i.e.:

```bash
(⎈ minikube:tenant-dev-foo-mattysoup)➜  kubeapps git:(master) ✗ KUBECONFIG=~/Downloads/cw-kubeconfig k get ns
Error from server (Forbidden): namespaces is forbidden: User "token-A2fFgBd4EuU8u2sCNVve" cannot list resource "namespaces" in API group "" at the cluster scope
(⎈ minikube:tenant-dev-foo-mattysoup)➜  kubeapps git:(master) ✗ KUBECONFIG=~/Downloads/cw-kubeconfig k get ns tenant-dev-foo-mattysoup
NAME                       STATUS   AGE
tenant-dev-foo-mattysoup   Active   3h18m
```
So as you can see, `k get ns` errors while `k get ns my-namespace` succeeds.

In production we do not yet have our API gateway/auth proxy deployed and as such the browser appears not to make this `GET /api/clusters/default` request whatsoever and do not see the error.

When I visit a "namespaced route", e.g. https://apps.coreweave.test/#/c/default/ns/tenant-dev-foo-mattysoup/apps, I do not see this warning.

![image](https://user-images.githubusercontent.com/2043821/122118838-84e23c00-cdf6-11eb-89f4-d817fc2905b6.png)

**Comment by mecampbellsoup** [comment]
This appears to be fixed now in chart version 7.X.X 👍 

However we need to exploit the ns whitelist feature @goock did in #2671 so this PR fix needs merged ASAP please: https://github.com/kubeapps/kubeapps/pull/2981

**Comment by mecampbellsoup** [comment]
UPDATE: resolving this issue _depends on_ using #2671 as it sets the list of namespaces that can be accessed via our auth header (in our case `x-consumer-permissions`).

The following is running version 7.1.0 of the kubeapps chart.

![image](https://user-images.githubusercontent.com/2043821/122124110-d261a780-cdfc-11eb-98fb-8ee18f169b4b.png)

**Comment by absoludity** [comment]
So afaict, the error message is correct, and the actual cause of the error will be fixed by your chart change at bitnami/charts#6675 ? I'm moving this into our Done lane, but feel free to move it back if that's not the case. Also, once you've verified the new chart, you can close this issue.
